src/main/resources/application.yaml

```diff
@@ -242,6 +242,10 @@ document:
       prl_c47a_filename: C47ACreateOrder.pdf
       prl_c47a_draft_template: PRL-ORDERS-C47A-DRAFT.docx
       prl_c47a_draft_filename: C47ACreateOrderDraft.pdf
+      prl_n117_draft_template: PRL-ORDERS-N117-DRAFT.docx
+      prl_n117_draft_filename: N117CreateOrderDraft.pdf
```

Comment:
Can we rename the file name as per the order label name which we display in order?